### PR TITLE
feat: improve version handling

### PR DESCRIPTION
**Pull request by EricRibeiro**
### Checklist

- [x] All new jobs, commands, executors, parameters have descriptions
- [x] Examples have been added for any significant new features
- [x] README has been updated, if necessary

### Motivation, issues

Closes:

- #49 
- #46 
- #45 
- #19

### Description

This PR changes the default CLI version to `latest`. Users can follow the existing process of specifying it via the `version` parameter if they wish to install a custom version.

It also adds several tests for different platforms, including Alpine. The latter is now officially supported once the user installs the necessary dependencies.

### CODE REVIEW

File: src/commands/install.yml [hunk 1]
```diff
@@ -7,15 +7,31 @@ description: |
...
+    description: >
+      This parameter allows you to selectively run or skip the command depending on the status of the job.
+      For more details, see: https://circleci.com/docs/configuration-reference/#the-when-attribute
 
 steps:
   - run:
       name: Install latest gcloud CLI version, if not available
       environment:
-        ORB_VAL_VERSION: <<parameters.version>>
+        ORB_VAL_VERSION: << parameters.version >>
```
Comment (Jaryt) [inline src/commands/install.yml:34]: Maybe no_output_timeout belongs for this one, but keep `when` for the next?

Comment (EricRibeiro) [reply src/commands/install.yml:34]: > keep when for the next?

The next step? If so, I don't see why we would need another step in this command.

Comment (Jaryt) [reply src/commands/install.yml:34]: Maybe `when` doesn't exactly make sense in this context then.

**Review by Jaryt — CHANGES_REQUESTED** [review]

**Review by Jaryt — APPROVED** [review]
Lookin good!

**Pull request merged**